ecosystem/typescript/sdk/package.json

```diff
@@ -32,31 +33,36 @@
     "Aptos Labs",
     "Move"
   ],
+  "dependencies": {
+    "axios": "^0.27.2",
+    "buffer": "^6.0.3",
+    "eslint-config-prettier": "^8.5.0",
+    "form-data": "^4.0.0",
+    "js-sha3": "^0.8.0",
+    "lodash": "^4.17.21",
```

Comment:
Do you need lodash as a whole? You probably just need to load a subset of packages.

Reply:
I only need isEqual, I didn't realise you could import parts of it. I'll investigate that.